resources/newssite/news-next/dist/benchmark-connector.mjs

```diff
@@ -0,0 +1,97 @@
+/*
+    Note: This will eventually come from a separate package and should be minified.
+*/
+window.name = "news-site-next";
+window.version = "1.0.0";
+
+/** **********************************************************************
+ * Params
```

Comment:
@flashdesignory I don't see the following logic used in the main runner. Since it also reads params from the URL, would it be possible to share some code here instead?

Reply:
we could - the params.mjs is slightly different, but I can take a look at both and see if I can clean up and reuse